input/pagecontent/2-9-GetCurrentContext.md

```diff
@@ -25,7 +25,7 @@ Field | Optionality | Type | Description
 
 ### Content Sharing Support
 
-If a Hub supports content sharing, the Hub returns the current content in a `content` key in the `context` array.  There SHALL be one and only one `Bundle` resource which SHALL have a `type` of `collection`.  No entry in the `Bundle` SHALL contain a `request` attribute.  The `Bundle` SHALL contain no entries if there is no content associated with the current context.
+If a Hub supports content sharing, the Hub returns the current content in a `content` key in the `context` array.  There SHALL be one and only one `Bundle` resource which SHALL have a `type` of `collection`. The `Bundle` resource SHALL conform to the [FHIR Bundle](https://build.fhir.org/foundation-module.html) specification.

```

Comment:
I think that this is probably the intended url -- https://hl7.org/fhir/R4/bundle.html#transaction

Also, it's better to reference a published version of the base FHIR spec, instead of the ci build. It's questionable here if we should be referenced R4 vs just the latest FHIR.